quoridorui/src/main/java/com/harush/zitoon/quoridor/ui/controller/MainMenuController.java

```diff
@@ -17,15 +17,17 @@
 public class MainMenuController extends AbstractController {
 	
 	@FXML
-	private Button exitButton;
+	private Button button;
```

Comment:
This actually is the exit button. Why do you think changing it to a less clear name is better?